features/cellular/framework/AT/AT_CellularContext.cpp

```diff
@@ -1065,3 +1065,16 @@ void AT_CellularContext::set_disconnect()
     _is_connected = false;
     _device->cellular_callback(NSAPI_EVENT_CONNECTION_STATUS_CHANGE, NSAPI_STATUS_DISCONNECTED, this);
 }
+
+void AT_CellularContext::is_prefer_ipv6(char *ip, size_t ip_size, char *ip2, size_t ip2_size)
```

Comment:
This wouldnt be needed if CGCONTRDP's local address is checked, instead of quering the info from the stack.

Reply:
@mirelachirica please re-review